lib/diagnostic.js

```diff
@@ -32,7 +38,9 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
-
+db.houses.update({house: 'Stark'}, {$set: {'members': ['Ned','Arya','Sansa']}});
```

Comment:
The syntax is correct, but the query is not.  There are no documents with `house` as a key.